packages/icons/src/svg/products/qlik-colored.svg

```diff
@@ -1,15 +1,8 @@
-<svg
-    xmlns="http://www.w3.org/2000/svg"
-    viewBox="0 0 16 16">
-    <g>
-        <path
-            d="m 14.03928,15.064548 -2.01405,-1.736085 a 7.0939155,7.5573167 0 1 1 1.613681,-2.415786 l 2.148074,1.851694 c 0,0 0.318707,0.275052 0.06068,0.615436 l -1.230683,1.620099 c 0,0 -0.258029,0.340383 -0.577699,0.06463 M 7.9984014,12.383283 a 4.3582892,4.6429891 0 1 0 -1.667266,-9.1145251 4.3582892,4.6429891 0 0 0 1.667266,9.1145251"
-            fill="#009c4d"
-            stroke-width="0.847154"
-        />
-        <path
-            d="m 3.740584,7.8113626 a 3.404,3.404 0 1 0 6.808,0 3.404,3.404 0 0 0 -6.808,0"
-            fill="#54565b"
-        />
-    </g>
-</svg>
\ No newline at end of file
+<svg viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path fill-rule="evenodd" clip-rule="evenodd"
+        d="m 12.02985,7.6825299 c 0,2.6068651 -2.0495995,4.7204131 -4.5784999,4.7204131 -2.5287701,0 -4.5787401,-2.113548 -4.5787401,-4.7207744 0,-2.6069164 2.04997,-4.7200212 4.5787401,-4.7200212 2.5286504,0 4.5784999,2.1131048 4.5784999,4.7203825 z m 3.80795,5.9006461 -2.08505,-1.803768 c 0.7272,-1.185807 1.1498,-2.5901941 1.1498,-4.0968781 C 14.90255,3.4395875 11.5666,0 7.4513501,0 3.33619,0 0,3.4395875 0,7.6821686 0,11.92475 3.33619,15.364802 7.4513501,15.364802 c 1.59645,0 3.0749999,-0.51902 4.2876499,-1.400413 l 2.2234,1.923406 c 0,0 0.3305,0.285833 0.60785,-0.05492 l 1.3208,-1.622916 c 0,0 0.2775,-0.3408 -0.05325,-0.626787 z"
```

Comment:
I was wondering what is the diff between m and M, look 
![image](https://github.com/Talend/ui/assets/19857479/7492d768-0297-4d08-ab18-72408a7a3672)